project/users/management/commands/create_groups.py

```diff
@@ -0,0 +1,36 @@
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import Group, Permission
+from django.contrib.contenttypes.models import ContentType
+from project.partners.models import Partner
+from project.program.models import Activity, Presenter
+
+PERMISSIONS = ['view', 'edit', 'add', 'delete']
+permissions = {'media': ['partner'], 'fr': ['partner'],
+               'experience': ['activity', 'presenter'], 'graphics': [],
+               'venue': [], 'speakers': ['presenter']}
+
+
+class Command(BaseCommand):
+    help = 'Create group permissions.'
+
+    def handle(self, *args, **kwargs):
+        for group in permissions.keys():
+            new_group, created = Group.objects.get_or_create(name=group)
+            for model in permissions[group]:
+                if (model == 'partner'):
+                    ct = ContentType.objects.get_for_model(Partner)
+                elif (model == 'activity'):
+                    ct = ContentType.objects.get_for_model(Activity)
+                else:
+                    ct = ContentType.objects.get_for_model(Presenter)
+                for permission in PERMISSIONS:
+                    name = 'Can {} {}'.format(permission, model)
+                    try:
```

Comment:
[`get_or_create`](https://docs.djangoproject.com/en/2.1/ref/models/querysets/#get-or-create)

Reply:
I guess you are referring to Permissions, right?

Reply:
I think I am missing something here. Could you please elaborate?

Reply:
```python
try:
    model_add_perm = Permission.objects.get(name=name,
                                            content_type=ct)
except:
    model_add_perm = Permission.objects.create(name=name,
                                               content_type=ct)
```
can be simplified to

```python
model_add_perm, _ = Permission.objects.get_or_create(name=name,
                                                     content_type=ct)
```

Reply:
Oh, yes, sorry.
I swear I tried that and I was getting an error. What happened was that the error was something that I solved later on and I stayed with the wrong impression afterwards.
Fixed.